scss/components/_award.scss

```diff
@@ -20,14 +20,36 @@
   th:first-child {
     width: 16px;
   }
+}
+
+.civiawards__award__stages,
+.civiawards__award__review-fields {
+  [type='checkbox'] {
+    margin-top: -5px;
+  }
 
   tbody {
     tr {
       cursor: pointer;
     }
   }
+}
 
-  [type='checkbox'] {
-    margin-top: -5px;
+.civiawards__award__review-fields--selected {
+  margin: 0 20px 20px !important;
+  width: calc(100% - 40px);
+
+  tbody {
+    tr {
+      cursor: auto;
+    }
   }
 }
+
+.civiawards__award__add-review-fields {
+  margin: 20px;
```

Comment:
do we have variables for this `20px` margin? Can you switch to this variable everywhere else?